pom.xml

```diff
@@ -29,13 +29,14 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.build.timestamp.format>yyyy-MM-dd-HH-mm</maven.build.timestamp.format>
+    <gcloud.api.version>0.13.0-beta</gcloud.api.version>
```

Comment:
@meltsufin I just tried 1.0.0, but it failed with this issue: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1994